src/common/utility.cpp

```diff
@@ -595,4 +602,19 @@ QByteArray Utility::conflictFileBaseName(const QByteArray &conflictName)
     return "";
 }
 
+QString Utility::sanitizeForFileName(const QString &name)
+{
+    const auto invalid = QStringLiteral("/?<>\\:*|\"");
+    QString result;
+    result.reserve(name.size());
+    for (const auto c : name) {
```

Comment:
you did not like the use of std::remove_if?

Reply:
Yes, I didn't like it at all. I thought `remove_if` followed by `truncate(distance(begin, newend))` looked way worse than this